Aztec/Classes/NSAttributedString/Conversions/AttributedStringParser.swift

```diff
@@ -899,6 +899,14 @@ private extension AttributedStringParser {
             element.updateAttribute(named: key, value: .string(finalValue))
         }
 
+        if let linkText = attachment.linkURL?.absoluteString {
+            let hrefValue = Attribute.Value(withString: linkText)
+            let hrefAttribute = Attribute(name: "href", value: hrefValue)
```

Comment:
I'm not very familiar with the Aztec codebase, but `"href"` here feels like something we could use a constant for, instead of using string literals.

Reply:
You're right, and I see there's already a `HTMLLinkAttribute` enum with a constant for that. I'll change it.